.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,24 +1,28 @@
 ---
 name: Bug report
 about: Found a bug? Let us know!
-title: ''
+title: 'bug: <description>'
 labels: bug
-assignees: ''
-
+assignees: 'robdodson'
```

Comment:
Would it be ok to leave this unassigned? I try to only set something assigned to me when I intend to work on it. It makes it much easier to use the GitHub search UI to figure out what I need to work on next.

With other folks (ewa, michael s.) ramping up soon, I'll probably start doing a regular triage for bugs and feature requests and assign them.